comments/classified.csv

```diff
@@ -1,83 +1,78 @@
-text,isnegative,importance
-_holy crap you actually did it_,0,0.5
-_technically_ yes.,0,0.5
-- [ ] Investigate caching TypeConverters for properties and types. I'd recommend going for a sourcegen solution and generate a switch instead of a dictionary to avoid allocation bump,0,0.5
-- [ ] Investigate why ColorTypeConverter and EasingTypeConverter are whitelisted,0,0.5
-- [x] - #code is having scaling issues (migrie: ✔️ I think; @github ✅ i can't get anything weird to happen ),0,0.5
-- [x] - #code needs to account for scaling (migrie: ✔️),0,0.5
-- [x] - Can we completely hide all pixel stuff into the renderer for the fake font scaling - Not likely in this PR,0,0.5
```

Comment:
There are way too many lines changed in this file.

Instead of sorting it alphabetically, can you just add the new row to the end? I can't tell what you changed the diff is so large. Make the diff small as possible.